.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
`--require rails_helper`

It would load everything since rspec_helper is included into rails_helper

Reply:
Fixed.